app/uk/gov/hmrc/cataloguefrontend/connector/TeamsAndRepositoriesConnector.scala

```diff
@@ -42,6 +42,8 @@ object RepoType {
 
   val values: List[RepoType] = List(Service, Library, Prototype, Test, Other)
 
+  val dependencyExplorerValues: List[RepoType] = List(Service, Library, Test, Other)
```

Comment:
dependency explorer doesn't need to pollute the `RepoType` model. Just add `RepoType.values.remove(RepoType.Prototype)` to DependencyExplorer, which will make it clear what it supports too